buildlib/azure-pipelines.yml

```diff
@@ -140,6 +140,7 @@ stages:
       - job: publish_jucx
         displayName: Publish JUCX SNAPSHOT artifact to maven central
         container: fedora
+        #condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
This one is tricky: this pipeline runs two times, first for the PR itself and then for a merge commit (not counting PR fixes).
I guess this step is required only for a merge commit. @yosefe @petro-rudenko WDYT?

Reply:
Yes, let's test for now in PR, after uncomment and it should run only when master branch updated.

Reply:
Another question: When we build rc builds do we run pipeline-release? Because release needs to trigger just once, when it's a final release.

Reply:
The release pipeline is triggered only for tag creation. For every RC we will have a release in Maven too.

Reply:
Maven can do release just once.

Reply:
Is it possible to have some parameter in AZP like `finalrelease` and set it to tru when it's a final release

Reply:
Can we post RCs to Maven central? Versions are different: `v1.8rc1`, `v1.8rc1` and `v1.8`. The last one is the final. Optionally we can check tag name for this `rc` suffix.